applications/typechain-polkadot-follow-up.md

```diff
@@ -0,0 +1,221 @@
+
+# W3F Grant Proposal
+
+- **Project Name:** [Typechain-Polkadot](https://github.com/Supercolony-net/typechain-polkadot)
+- **Team Name:** [SuperColony](https://github.com/Supercolony-net)
+- **Payment Address:** ERC 20 ADDRESS:
+  0xE1B19cE32866cDE87F8f59C7C2C5f90E093A6942
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+Nowadays, when technologies are growing faster and faster, we should think about optimizations of different routine processes and making older stuff better. One of these optimizations is to make code typesafe that will be flexible in different situations.
+
+When a smart contract is being written, front-end developer receives file representation of it in the format called Application Binary Interface (ABI). One ABI per each contract, new ABI for every update of a contract.
+
+Information about how to interact with a contract (methods names, arguments & returns types etc.) is included in this ABI file. It is not quite human-readable, so extraction of that information becomes a challenge. We need to have correct type definitions for each contract in TypeScript.
+
+Interaction with blockchain is done with polkadot.js library, which only has abstract definitions for the contract in use, thus users' code cannot be typesafe. And Typechain-Polkadot can change it.
+
+This Grant is a follow-up to the previous grant, [W3F Grant Proposal](https://github.com/w3f/Grants-Program/pull/992).
+We have decided to add new tool that will help developers to easy integrate typechain to their projects, and it will be also compatible with Redspot (if redspot was used to compiler project, you can just left most of things).
+We have some updates in our team, so check it out. And also we made more work on Milestone 1 than we thought, so now it'll be easier to add new features to the project, so we considered to reestimate our time.
+
+### Usage of Typechain-compiler
+
+```bash
+    npx typechain-compiler --config config.json
+```
+
+Also you can set some additional arguments like `--noCompile`, `--noTypechain`, `--release`
+
+Config interface will be something like this:
+```typescript
+export interface Config {
+    projectFiles: string[]; // Path to all project files, everystring in glob format
```

Comment:
What do you mean by "glob format"?

Reply:
"Globs" are the patterns you type when you do stuff like ls *.js on the command line, or put build/* in a .gitignore file